tests/api_test/api_test.cpp

```diff
@@ -56,6 +56,8 @@ _program_load_helper(
     ebpf_result_t result;
     const char* log_buffer = nullptr;
     result = ebpf_program_load(file_name, program_type, nullptr, execution_type, object, program_fd, &log_buffer);
+    if (result != EBPF_SUCCESS)
+        printf("Logbuffer: %s\n", log_buffer);
```

Comment:
Another printf to replace with a REQUIRE.  (Either in this PR or in a separate one if you file a github issue.)

Reply:
Removed printf. See https://github.com/microsoft/ebpf-for-windows/pull/336#discussion_r668854535.